app/views/docs/functions.phtml

```diff
@@ -503,14 +503,64 @@ static RuntimeResponse &main(const RuntimeRequest &req, RuntimeResponse &res) {
 
 <p><a href="/docs/server/functions#functionsCreateDeployment" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> You can also create new code deployments using the Appwrite server API</a></p>
 
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a typical Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── src
+    └── main.js
+</code></pre>
+</div>
+
+<p>First, package your code files into the <code>.tar.gz</code> format with the tar command:</p>
```

Comment:
We **MUST** make it clear where this command should run. Which is inside the folder. Not outside, not inside `src`.

We do some folder location explaining in migration guide. That might be a good place to copy some text from: https://appwrite.io/docs/upgrade

Reply:
@Meldiron They never read these, which is why I tell them to specify the files so they'd never have this issue :P